vaadin-spring-boot-starter/pom.xml

```diff
@@ -76,7 +76,7 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.4.12</version>
+            <version>1.4.14</version>
```

Comment:
Copy from the closed PR: Got another thing for you :) You should probably also add the other logback-* dependencies, otherwise you get a mismatch which is hard to debug: https://github.com/qos-ch/logback/issues/744#issue-2017777332